packages/babel-parser/src/tokenizer/state.js

```diff
@@ -137,10 +137,15 @@ export default class State {
   // escape sequences must not be interpreted as keywords.
   containsEsc: boolean = false;
 
-  // This property is used to throw an error for
-  // an octal literal in a directive that occurs prior
-  // to a "use strict" directive.
-  octalPositions: number[] = [];
+  // This property is used to track the following errors
+  // - StrictNumericEscape
+  // - StrictOctalLiteral
+  //
+  // in a literal that occurs prior to/immediately after a "use strict" directive.
+
+  // todo(JLHwung): set strictErrors to null and avoid recording string errors
+  // after a non-directive is parsed
```

Comment:
Do you want to do this here or in a separate PR?

Reply:
I prefer a separate PR since alternatively, todo is pronounced as never-do. 😛 
Legacy octal or numeric escapes are discouraged, I feel like it is okay to leave it as-is.